plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1174,10 +1176,32 @@ private void executeExpireSnapshots(ConnectorSession session, IcebergTableExecut
                 IcebergSessionProperties.EXPIRE_SNAPSHOTS_MIN_RETENTION);
 
         long expireTimestampMillis = session.getStart().toEpochMilli() - retention.toMillis();
+        TrinoFileSystem fileSystem = fileSystemFactory.create(session);
+        Set<String> pathsToDelete = new HashSet<>();
+        Consumer<String> deleteFunc = path -> {
```

Comment:
`deleteFunc` is Iceberg API parameter name, but we have a different naming convention, e.g. we don't use abbreviations like this.

let's call it `deleteFunction`

Reply:
eh i always forget about not using abbreviations